uk.co.compucorp.civicrm.hrleaveandabsences/tests/phpunit/CRM/HRLeaveAndAbsences/Service/AbsenceTypeTest.php

```diff
@@ -161,4 +161,17 @@ public function testAbsenceTypeHasEverBeenUsedReturnsFalseWhenAbsenceTypeIsNotLi
     $absenceTypeID = 1;
     $this->assertFalse($this->absenceTypeService->absenceTypeHasEverBeenUsed($absenceTypeID));
   }
+
+  public function testAbsenceTypeGetCategories() {
```

Comment:
No need to add AbsenceType in the test name since the class namespace already has the AbsenceType keyword.. `testGetCategories` should be enough.